pkg/testutil/schema_helper.go

```diff
@@ -24,7 +24,38 @@ import (
 	ackmodel "github.com/aws-controllers-k8s/code-generator/pkg/model"
 )
 
+// TestingModelOptions contains optional variables that are passed to
+// `NewModelForServiceWithOptions`.
+type TestingModelOptions struct {
+	// The CR API Version. Defaults to v1alpha1
+	APIVersion string
+	// The generator config file. Defaults to generator.yaml
+	GeneratorConfigFile string
+	// The AWS API version. Defaults to 00-00-0000
+	SDKAPIVersion string
```

Comment:
nit: This is not the `SDKAPIVersion`, it's a versioned date string related to the API backend format (or something like that). It should not be confused for the AWS SDK version.

Reply:
Yepp changed to `ServiceAPIVersion` to avoid confusion with AWS SDK Version.

Reply:
In the `api-2.json` it is called `apiVersion` https://github.com/aws/aws-sdk-go/blob/main/models/apis/kafka/2018-11-14/api-2.json#L3